src/main/java/net/sf/jabref/gui/event/Event.java

```diff
@@ -0,0 +1,21 @@
+package net.sf.jabref.gui.event;
+
+/**
+ * Represents a medium which can be used
+ * to pass objects on event occurrence.
+ */
+
+public class Event {
```

Comment:
Why not use https://docs.oracle.com/javase/7/docs/api/java/util/EventObject.html for this?